packages/web-components/src/radio-group/README.md

```diff
@@ -0,0 +1,127 @@
+
+### Use Case
+
+Radio group allows the user to be presented with a list of all the options visible which can facilitate the comparison of choice.
+
+<br />
+
+## Class: `RadioGroup`
+
+<br />
+
+### **Component Name**
+
+<br />
+
+`fluent-radio-group`
+
+<br />
+
+### **Fields**
+
+| Name          | Privacy | Type                     | Default      | Description                                                                                           |
+| ------------- | ------- | ------------------------ | ------------ | ----------------------------------------------------------------------------------------------------- |
+| `disabled`    | public  | `boolean`                | `false`      | Disables the radio group and child radios.                                                            |
+| `name`        | public  | `string`                 |              | The name of the radio group. Setting this value will set the name value for all child radio elements. |
+| `value`       | public  | `string`                 |              | The value of the checked radio.                                                                       |
+| `orientation` | public  | `horizontal \| vertical` | `horizontal` | The orientation of the group                                                                          |
+| `childItems`  | public  | `HTMLElement[]`          |              | The orientation of the group                                                                          |
+
+<br />
+
+### **Methods**
+
+| Name              | Privacy   | Description | Parameters | Return | Inherited From |
+| ----------------- | --------- | ----------- | ---------- | ------ | -------------- |
+| `nameChanged`     | protected |             |            | `void` |                |
+| `valueChanged`    | protected |             |            | `void` |                |
+| `templateChanged` | protected |             |            | `void` |                |
+| `stylesChanged`   | protected |             |            | `void` |                |
+
+<br />
+
+### **Events**
+
+| Name     | Type | Description                                          |
+| -------- | ---- | ---------------------------------------------------- |
+| `change` |      | Fires a custom 'change' event when the value changes |
+
+<br />
+
+### **Attributes**
+
+| Name          | Field       | Inherited From |
+| ------------- | ----------- | -------------- |
+| `disabled`    | disabled    |                |
+| `named`       | name        |                |
+| `value`       | value       |                |
+| `orientation` | orientation |                |
+
+<br />
+
+### **Slots**
+
+| Name  | Description                       |
+| ----- | --------------------------------- |
+|       | The default slotted content       |
```

Comment:
These aren't default slotted content - this is the default slot which expects radios